flashinfer/gemm.py

```diff
@@ -1240,3 +1249,174 @@ def group_deepgemm_fp8_nt_groupwise(
     )
 
     return out
+
+
+class UIDs(Enum):
+    """UIDs for CUDNN graph tensors"""
+
+    A_UID = 0
+    B_UID = 1
+    SCALE_UID = 2
+    O_UID = 3
+
+
+def _check_cudnn_availability():
+    """Check if cuDNN is available and raise exception if not."""
+    if not CUDNN_AVAILABLE:
+        raise RuntimeError(
+            "cuDNN is not available. Please install cuDNN to use FP8 GEMM functions. "
```

Comment:
Can you add how to install? 
`pip install nvidia-cudnn-cu12 nvidia-cudnn-frontend`

Reply:
We can specify them as package level dependencies: 
https://github.com/flashinfer-ai/flashinfer/blob/f153369384db9298f52b00d8e0fc53cb0d1f0f49/setup.py#L54

Currently the only bottleneck is the aarch64 wheels availability.

Reply:
can I remove the `_check_cudnn_availability` after update the setup.py?